SynRD/synthesizers/synthesizer.py

```diff
@@ -206,6 +207,26 @@ def sample(self, n):
         df = self.synthesizer.sample(n)
         df = self._unslide_range(df)
         return df
-    
-class AIMSynthesizer(Synthesizer):
-    pass
\ No newline at end of file
+
+class AIMTSynthesizer(Synthesizer):
```

Comment:
I'm not sure what else should be changed. I checked the API of AIM, seems very similar to the MST. Please let me know if there is anything I should additionally check.